pep-0696.rst

```diff
@@ -190,38 +196,52 @@ default to the type of ``start`` and step default to ``int | None``.
 
    class slice(Generic[StartT, StopT, StepT]): ...
 
+   reveal_type(slice)  # type is type[slice[StartT = int, StopT = StartT, StepT = int | None]]
    reveal_type(slice())                        # type is slice[int, int, int | None]
    reveal_type(slice[str]())                   # type is slice[str, str, int | None]
    reveal_type(slice[str, bool, timedelta]())  # type is slice[str, bool, timedelta]
 
+   T2 = TypeVar("T2", default=DefaultStrT)
+
+   class Foo(Generic[DefaultStrT, T2]):
+       def __init__(self, a: DefaultStrT, b: T2) -> None: ...
+
+   reveal_type(Foo(1, ""))  # type is Foo[int, str]
+   Foo[int](1, "")          # Invalid: Foo[int, str] cannot be assigned to self: Foo[int, int] in Foo.__init__
+   Foo[int]("", 1)          # Invalid: Foo[str, int] cannot be assigned to self: Foo[int, int] in Foo.__init__
+
 When using a ``TypeVarLike`` as the default to another ``TypeVarLike``.
 Where ``T1`` is the default for ``T2`` the following rules apply.
 
+.. note::
+
+   ``TypeVarTuple``\s as defaults are not supported due to
+:ref:`scoping-rules` not allowing usage of ``TypeVarLikes`` from outer
+scopes, multiple ``TypeVarTuple``\s not being able appear in the same
+type parameter lists to a class and function defaults not being
+supported.
```

Comment:
```suggestion
   :ref:`scoping-rules` not allowing usage of ``TypeVarLikes`` from outer
   scopes, multiple ``TypeVarTuple``\s not being able appear in the same
   type parameter lists to a class and function defaults not being supported.
```

I forgot to explicitly include indenting the rest of the lines in my actual suggestion, sorry—that's why, predictably, you're seeing a build failure.